sqlx-cli/src/lib.rs

```diff
@@ -13,6 +13,14 @@ mod prepare;
 pub use crate::opt::Opt;
 
 pub async fn run(opt: Opt) -> anyhow::Result<()> {
+    let path = env::current_dir()?.join("Cargo.toml");
+    if !path.exists() {
+        return Err(anyhow!(
```

Comment:
`anyhow` provides a `bail!()` macro that covers this idiom:

```rust
if !path.exists() {
  bail!("failed to read ...", path);
}
```

Reply:
Thanks! Updated.